ocaml/testsuite/tests/typing-missing-cmi/test.compilers.reference

```diff
@@ -2,7 +2,7 @@ File "main.ml", line 1, characters 8-11:
 1 | let _ = A.a = B.b
             ^^^
 Error: This expression has type M.a but an expression was expected of type
-         ('a : value)
-       M.a has layout any, which is not a sublayout of value.
+         ('a : '_representable_layout_1)
```

Comment:
This is ugly. Can we do better? Or maybe this happens only for missing cmis, so maybe it's OK?

Reply:
I was kind of hoping your histories patch would magically make this better when merged.  Is that too optimistic?

Reply:
(Just resolve after you read this.)